### DISCUSSION TITLE
Utilité pseudonymisation

### DISCUSSION THREAD

**Discussion by merouanebouderbala** [danymat/INSAnonym-utils#60] [Q&A, Bug report & Support] [answered]
Bonjour, en utilisant l'algorithme de pseudonymisation, j'ai remarqué que l'id est remplacé par un pseudonyme pour toute les entrés, mais en soumettant, on peut observer une perte d'utilité, ne devrait-elle pas rester à 1 ?

**Comment by danymat** [comment, marked as answer]
Bonjour, en effet la perte d'utilité peut être expliquée par quelques métriques qui ne font pas du ligne à ligne, mais se basent par identifiant pour faire leurs comparaisons: Je pense notamment aux métriques utility_POI et utility_tuile. 
Donc ces derniers devraient normalement faire descendre le score d'utilité.

Vous pouvez essayer mes dires en faisant tourner les métriques unes par unes: https://github.com/danymat/INSAnonym-utils/blob/main/docs/insanonym-metrics.md.
